homework19/src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+	display: flex;
```

Comment:
Jesi li ovo probao malo na svim browser-ima: Firefox, Chrome, IE? Cim ne vidim nikakve specificnosti sto se toga tice pretpostavljam da nisi. Takodjer jeli ovo responsive design (posto ne vidim nista da to podrzi ;)).